components/header/index.tsx

```diff
@@ -24,6 +31,7 @@ const Header = (): JSX.Element => {
 			</HeaderWrapper>
 			<LeaderboardWrapper as='header'>
 				<Title>Leaderboard</Title>
+				<TimeframeSelectorDesktop />
```

Comment:
Does this component get rendered on Mobile? I'm surprised to see these in separate places

Reply:
It doesn't get rendered on mobile.
The issue is that on desktop the timeframe selector will be displayed on the header, but on mobile it will be displayed on the main.

Reply:
It seems to become hidden, but I do see it in the HTML stack and with React dev tools still. As opposed to the mobile selector modal which disappears from the HTML when closed

Don't worry too much about it for now, its a small performance boost at best